docs/develop/support-ie-11.md

```diff
@@ -48,7 +50,34 @@ Importing the entire core-js library will import all core-js features. You can a
 
 For a sample add-in that uses core.js, see [Word Add-in Angular2 StyleChecker](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker).
 
-## Testing an add-in on Internet Explorer
+## Determine at runtime if the add-in is running in Internet Explorer
+
+Your add-in can discover if it is running in Internet Explorer by reading the [window.navigator.userAgent](https://developer.mozilla.org/docs/Web/API/Navigator/userAgent) property. This enables the add-in to either provide an alternate experience or gracefully fail. The following is an example. Note that Internet Explorer sends a string beginning with "Trident" as the value of userAgent.
+
+```javascript
+if (navigator.userAgent.indexOf("Trident") === -1) {
+
+    // IE is not the browser. Provide a full-featured version of the add-in here.
+
+} else {
+
+    // IE is the browser. So here, do one of the following: 
+    //  1. Provide an alternate experience that does not use any of the HTML5
+    //     features that are not supported in IE.
+    //  2. Enable the add-in to gracefully fail by putting a message in the UI that
+    //     says something similar to: 
+    //      "This add-in won't run in your version of Office. Please upgrade to 
+    //      either one-time purchase Office 2021 or to a Microsoft 365 account."          
+
+}
+```
+
+> [!IMPORTANT]
+> It is not usually a good practice to read the `userAgent` property. Be sure you are familiar with the article, [Browser detection using the user agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent), including the recommendations and alternatives to reading `userAgent`. In particular, if you are taking option 1 in the `else` clause above, consider using feature detection instead of testing for the user agent.
```

Comment:
```suggestion
> It is not usually a good practice to read the `userAgent` property. Be sure you are familiar with the article, [Browser detection using the user agent](https://developer.mozilla.org/docs/Web/HTTP/Browser_detection_using_the_user_agent), including the recommendations and alternatives to reading `userAgent`. In particular, if you are taking option 1 in the `else` clause above, consider using feature detection instead of testing for the user agent.
```

Reply:
In this case, we want readers to go to the English version for reasons explained just below.